plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 2.2.1
+version: 2.3.0
```

Comment:
Just to check the update to `Get XQL Query Results` action doesn't break down existing workflows in the staging environment? I don't think it should but since we are changing `results` to a new type do the references to this output persist and are there any values that we're previously returned in `results` which are now affected?

Reply:
It shouldn't break. The previous type for the results was `object`, and the customers had to manually catch fields that they needed, for example in `Loop` component

Reply:
Ok that makes sense thanks. Just to check there is nothing outside of data and stream_id that could be accessed previously?